http2-tracing/Readme.md

```diff
@@ -0,0 +1,27 @@
+# gRPC client & server
```

Comment:
I think this readme could be a bit more spelled out. Take a look at https://github.com/pixie-io/pixie-demos/tree/main/custom-k8s-metrics-demo for inspiration. At the very least, I think this file should have:
- a title like `HTTP/2 Tracing with eBPF Uprobes`
- a line at the top like: [Debugging with eBPF Part 4: Tracing full body HTTP/2 request/responses](https://blog.px.dev/ebpf-http2-tracing) is the accompanying blog post for this demo. We can update the exact link once the blog post goes to PR form.
- a Prerequisites section
- a Usage section